server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -184,6 +184,17 @@ protected MManager() {
           MTREE_SNAPSHOT_THREAD_CHECK_TIME,
           TimeUnit.SECONDS);
     }
+
+    if (config.getSyncMlogPeriodInMs() != 0) {
+      timedForceMLogThread =
```

Comment:
if config.getSyncMlogPeriodInMs() == 0, what will we do ?

Reply:
Sync mlog after each refreshment, you can see it in the constructors of MLogWriter.